### ISSUE TITLE
Stuttering Dev Mod when set Flight Plan (Custom AP/ FADEC)

### ISSUE THREAD

**Issue by robro8** [flybywiresim/a32nx#4175]
**Mod Version**

Unstable Master Branch Build (with custom FBW / Autopilot / FADEC)

**Describe the bug**

is that known that in the 3D Cockpit start stuttering when i set the flight plan? 

**To Reproduce**

1.Start Flight at the Gate 308 from EDDM cold and dark
2.set flight plan manually or with simbrief
3.taxi to runway and take off

**Expected behavior**

the stutetring starst when the flight plan is set

**Actual behavior**

stuttering when Taxi and in flight

**References**

**Additional context**

Was this working before/when did the issue start occurring?
no stuttering before Custom AP and FADEC integrates
Is this a problem in the vanilla unmodded game?
no

![20210328_115930](https://user-images.githubusercontent.com/59294315/112748801-9ae21c80-8fbe-11eb-9f5f-f72f554e8ab1.jpg)

Discord username (if different from GitHub):

**BY REMOVING THIS LINE, I ATTEST THAT I HAVE FOLLOWED THE ISSUE TEMPLATE AND PROVIDED A TITLE, as well as searched for other possible duplicate issues. I acknowledge if this line is not removed and/or the issue template is not respected, my issue will be closed without warning**

**Comment by kutak11** [comment]
same here... very unstable FPS also during the flight after last updates... it is droping by 10fps periodically

**Comment by Benjozork** [comment]
So the upcoming custom flight plan manager is allegedly less stuttery, so I don't know the value of looking at this. Regardless, I'm gonna try to see if I can find anything.

**Comment by robro8** [comment]
> same here... very unstable FPS also during the flight after last updates... it is droping by 10fps periodically

yes, i have the same !!

**Comment by robro8** [comment]
is not closed

**Comment by SAS61Z** [comment]
Hey alltogether,
this is my first post here, unfortunately because I've got this issue too. Last week the stuttering (rhytmic stutters about every second) started at some point during pushback but went away few minutes after takeoff. In the following flights the stuttering didn't occur when I didn't use the flypad. Since I updated to the lastest version today that "trick" doesn't work anymore. Stuttering starts right after loading is complete (dark and cold at the gate). The Experimental version is unflyable, I changed to the Dev which it better, but performance is still worse than the Experimental two weeks ago.

I hope there will be a solution soon, I love flying this mod! ;-)
Big THANKS to the fbw crew you're doing an amazing job!
Best Max
